opensplice_cmake_module/cmake/Modules/FindOpenSplice.cmake

```diff
@@ -29,10 +29,15 @@ set(OpenSplice_FOUND FALSE)
 
 # check if provided OSPL_HOME is from an "official" binary package
 set(_ospl_home "$ENV{OSPL_HOME}")
-if(NOT "${_ospl_home} " STREQUAL " " AND
-    NOT EXISTS "${_ospl_home}/release.com")
+if(WIN32)
+  set(_ospl_release_file ${_ospl_home}/release.bat)
+else()
+  set(_ospl_release_file ${_ospl_home}/release.com)
+endif()
```

Comment:
Why not only store the filename in the variable especially since the logic is tighly coupled to the `ospl_home` path?